install.md

```diff
@@ -123,12 +123,30 @@ helm init --wait --service-account tiller
 
 ## Installing Kritis to your cluster
 
-Install the `resolve-tags` plug-in:
+Install the `resolve-tags` kubectl plugin and binary:
+
+## Mac OS X
+
+```shell
+curl -LO https://storage.googleapis.com/resolve-tags/latest/resolve-tags-darwin-amd64.tar.gz && \
+  RESOLVE_TAGS_DIR=$HOME/.kube/plugins/resolve && \
+  mkdir -p $RESOLVE_TAGS_DIR && tar -C $RESOLVE_TAGS_DIR -xzf resolve-tags-darwin-amd64.tar.gz && \
+  mv $RESOLVE_TAGS_DIR/resolve-tags-darwin-amd64 $RESOLVE_TAGS_DIR/resolve-tags && \
+  sudo cp $RESOLVE_TAGS_DIR/resolve-tags /usr/local/bin/
+```
+
+## Linux
 
 ```shell
-make install-plugin
+curl -LO https://storage.googleapis.com/resolve-tags/latest/resolve-tags-linux-amd64.tar.gz && \
+  RESOLVE_TAGS_DIR=$HOME/.kube/plugins/resolve && \
+  mkdir -p $RESOLVE_TAGS_DIR && tar -C $RESOLVE_TAGS_DIR -xzf resolve-tags-linux-amd64.tar.gz && \
+  mv $RESOLVE_TAGS_DIR/resolve-tags-linux-amd64 $RESOLVE_TAGS_DIR/resolve-tags && \
+  sudo cp $RESOLVE_TAGS_DIR/resolve-tags /usr/local/bin/
 ```
 
+For more information about the plugin, please visit our [documentation](cmd/kritis/kubectl/plugins/resolve/README.md).
```

Comment:
It may be unclear to the reader who "our" is, and what the documentation links to here. Perhaps: "For more information, see the [[resolve-tags plugin documentation]](...).

Reply:
done